google/auth/external_account.py

```diff
@@ -51,6 +52,22 @@
 # Cloud resource manager URL used to retrieve project information.
 _CLOUD_RESOURCE_MANAGER = "https://cloudresourcemanager.googleapis.com/v1/projects/"
 
+# Token url patterns
+_TOKEN_URL_PATTERNS = [
```

Comment:
Maybe define these inline since they are used once?

Reply:
Actually if you define them inline you don't have to pass them as a param and define in tests

Reply:
Yeap. Will copy the logic of the Java tests.